API/Utilities/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -43,12 +42,16 @@ public static bool IsWhitelisted(this ClaimsPrincipal claimsPrincipal) =>
         claimsPrincipal.FindFirst(OtrClaims.TokenType)?.Value;
 
     /// <summary>
-    /// Gets the <see cref="RateLimitOverrides"/> of the principle if available
+    /// Gets the custom rate limit of the principle if available
     /// </summary>
-    public static RateLimitOverrides? GetRateLimitOverrides(this ClaimsPrincipal claimsPrincipal)
+    public static int? GetRateLimitOverride(this ClaimsPrincipal claimsPrincipal)
     {
-        var raw = claimsPrincipal.FindFirst(OtrClaims.RateLimitOverrides)?.Value;
-        return string.IsNullOrEmpty(raw) ? null : RateLimitOverridesSerializer.Deserialize(raw);
+        if (!int.TryParse(claimsPrincipal.FindFirst(OtrClaims.RateLimitOverrides)?.Value, out var rlo))
```

Comment:
I would prefer not to use acronyms like this in local variables. `out var override`, `limit`, `rateLimitOverride`, etc. are more suitable options. The exception being `Func`/`Expression` arguments (lambdas).